tests/languages/d/issue2027.test

```diff
@@ -0,0 +1,47 @@
+string foo = "It's";
+foo = "It's";
+foo = "It's";
+
+string bar = `It's`;
+bar = `It's`;
+bar = `It's`;
+bar = `It's`;
+
+----------------------------------------------------
+
+[
+	["keyword", "string"],
+	" foo ",

```

Comment:
Is `foo` not a variable or what is it?

Reply:
Yes, it's a variable, why?

Reply:
Should it be highlighted as a variable?

Reply:
I don't think we should because:

1. We can't track the variable.
1. Almost no language (except PHP and other languages where variables have special syntax) highlights variables.
1. Wouldn't the code be a bit too colorful then?
1. Isn't the plaintext we don't highlight all just variables? So why highlight them?

Reply:
Ok, 2. convinces me; I knew we had it as a token, so I thought it was used more widely.